responsibleai/tests/rai_insights/test_served_model.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import json
+import os
+import pytest
+import random
+from responsibleai import RAIInsights, FeatureMetadata
+from tests.common_utils import (
+    create_tiny_forecasting_dataset, RandomForecastingModel)
+from unittest.mock import patch, Mock
+
+
+RAI_INSIGHTS_DIR_NAME = "rai_insights_test_served_model"
+
+
+# create a pytest fixture
+@pytest.fixture(scope="session")
+def rai_forecasting_insights_for_served_model():
+    X_train, X_test, y_train, y_test = create_tiny_forecasting_dataset()
+    train = X_train.copy()
+    train["target"] = y_train
+    test = X_test.copy()
+    test["target"] = y_test
+    model = RandomForecastingModel()
+
+    # create RAI Insights and save it
+    rai_insights = RAIInsights(
+        model=model,
+        train=train,
+        test=test,
+        target_column="target",
+        task_type='forecasting',
+        feature_metadata=FeatureMetadata(
+            datetime_features=['time'],
+            time_series_id_features=['id']
+        ),
+        forecasting_enabled=True)

```

Comment:
why is this extra forecasting_enabled flag needed if there is already a forecasting task type specified?

Reply:
Because it's still experimental. If you don't set the feature flag it won't work. The feature flag is not documented.